api/Plugin.json

```diff
@@ -20,7 +20,7 @@
             "version_added": "4"
           },
           "ie": {
-            "version_added": null
+            "version_added": "11"
```

Comment:
This should match https://github.com/mdn/browser-compat-data/pull/8002#pullrequestreview-555222775 when the correct version for `navigator.plugins` has been found. It might not be possible to reliably test an instance here though, since there might be no supported plugins.